trunk/src/app/srs_app_rtc_network.cpp

```diff
@@ -436,19 +436,19 @@ srs_error_t SrsRtcUdpNetwork::write(void* buf, size_t size, ssize_t* nwrite)
     return sendonly_skt_->sendto(buf, size, SRS_UTIME_NO_TIMEOUT);
 }
 
-SrsRtcTcpNetwork::SrsRtcTcpNetwork(SrsRtcConnection* conn, SrsEphemeralDelta* delta)
+SrsRtcTcpNetwork::SrsRtcTcpNetwork(SrsRtcConnection* conn, SrsEphemeralDelta* delta) : owner_(new SrsRtcTcpConn())
```

Comment:
Current branch seems rebased on the #4084 now, which already supported constructor the `SrsSharedResource` instance from a NULL pointer, so I guess this initialization can be improved by:
```suggestion
SrsRtcTcpNetwork::SrsRtcTcpNetwork(SrsRtcConnection* conn, SrsEphemeralDelta* delta) : owner_(NULL)
```

Reply:
No, reconsider this suggestion and let `SrsSharedResource` accept NULL pointer constructor, because 
https://github.com/ossrs/srs/blob/7b9c52b28357b49028eca82127d9d9c1855d5dd9/trunk/src/app/srs_app_conn.hpp#L163-L165

in this case, it's `owner_->interrupt()`.